odom_topic: "/summit_xl/odom"
cmd_topic_vel: "/summit_xl_control/cmd_vel"
position_source: "odom"
desired_freq: 50.0
target_frame: "/base_footprint"
command_type: "Twist"
d_lookahead_min: 3.0
d_lookahead_max: 3.5
d_dist_wheel_to_center: 0.3
desired_distance: 1.0
max_speed: 1.2
kr: 0.80 # default 0.20
